### PR TITLE
Update base image

### LINKED ISSUE #4394: Upgrade Python
Python 3.10 and 3.11 have brought several performance improvements. I wonder if we can't benefit of them by upgrading Python (we are using Python 3.9).

### PR DESCRIPTION
**Pull request by NickM-27**
## Proposed change

Updates the base image to debian bullseye

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code
- [ ] Documentation Update

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue: 

## Checklist

- [ ] The code change is tested and works locally.
- [ ] Local tests pass. **Your PR cannot be merged unless tests pass**
- [ ] There is no commented out code in this PR.
- [ ] The code has been formatted using Ruff (`ruff format frigate`)

### CODE REVIEW

**Comment by MarcA711** [comment]
@NickM-27 I will test it ASAP. Perhaps over the weekend. Thank you for notifying me.

File: docker/rocm/Dockerfile [hunk 1]
```diff
@@ -70,10 +70,10 @@ RUN apt-get -y install libnuma1
 WORKDIR /opt/frigate/
 COPY --from=rootfs / /
 
-COPY docker/rocm/requirements-wheels-rocm.txt /requirements.txt
-RUN python3 -m pip install --upgrade pip \
-    && pip3 uninstall -y onnxruntime-openvino \
-    && pip3 install -r /requirements.txt
+#COPY docker/rocm/requirements-wheels-rocm.txt /requirements.txt
```
Comment (hawkeye217) [inline docker/rocm/Dockerfile:73]: Can these commented lines just be removed?

Comment (NickM-27) [reply docker/rocm/Dockerfile:73]: I can add a note to the comment, but this is only temporary while I try to see if I can build one for py311

Comment (hawkeye217) [reply docker/rocm/Dockerfile:73]: Cool.

**Review by hawkeye217 — APPROVED** [review]

**Comment by NickM-27** [comment]
We can take another look and see if there is a way to get the YOLONAS notebook working.

> What model should I use? Are ultralytics models still not useable due to the license issues? Or do newer models like yolo11 have another license?

I am not entirely sure what the current rockchip options are, nothing has changed regarding ultralytics models, I believe there are some re-implementations of some of the YOLO model architectures without the AGPL3 license. 

> Moreover, I saw that there are new AI features in the new frigate version. Is there a way to accelerate this on different hardware (similar to detectors)?

Many platforms are supported, all of this is done through ONNX runtime. It looks like Rockchip has an ONNX runtime execution provider that is currently in preview state https://onnxruntime.ai/docs/execution-providers/community-maintained/RKNPU-ExecutionProvider.html

**Comment by MarcA711** [comment]
> We can take another look and see if there is a way to get the YOLONAS notebook working.

Someone mentioned [here](https://github.com/Deci-AI/super-gradients/issues/2068#issuecomment-2516166394) that one can download the weights manually. I will try to do this.

> I  am not entirely sure what the current rockchip options are, nothing has changed regarding ultralytics models, I believe there are some re-implementations of some of the YOLO model architectures without the AGPL3 license.

The ones that are supported best and are optimized for Rockchip are listed [here](https://github.com/airockchip/rknn_model_zoo?tab=readme-ov-file#model-support). However, almost every object detection model should work.

> Many platforms are supported, all of this is done through ONNX runtime. It looks like Rockchip has an ONNX runtime execution provider that is currently in preview state https://onnxruntime.ai/docs/execution-providers/community-maintained/RKNPU-ExecutionProvider.html

Unfortunately this is for the old generation of socs (not supported by Frigate and used by fewer people). Moreover, this has been in Preview state with no progress for years. I doubt that this will ever be finished.
Maybe I will find another way to implement this. I will have a look.

**Comment by NickM-27** [comment]
I imagine mobilenetv2 would be good to support at a minimum for a built in model and parity with other detectors. 

> Unfortunately this is for the old generation of socs (not supported by Frigate and used by fewer people). Moreover, this has been in Preview state with no progress for years.

Dang that's unfortunate

**Pull request merged**